packages/babel-helper-remap-async-to-generator/src/index.ts

```diff
@@ -21,13 +23,22 @@ const awaitVisitor = traverse.visitors.merge<{ wrapAwait: t.Expression }>([
     },
 
     AwaitExpression(path, { wrapAwait }) {
-      const argument = path.get("argument");
+      const { node } = path;
+      if (node.operation != null) {
+        node.argument = callExpression(
+          memberExpression(identifier("Promise"), node.operation, false, false),
```

Comment:
Nit: `false` is the default
```suggestion
          memberExpression(identifier("Promise"), node.operation),
```